File: 2st/Riyeon/조건 문자열.java
class Solution {
    public int solution(String ineq, String eq, int n, int m) {
        int answer = 0;
        if(eq.equals("=")){
            if(ineq.equals(">")) answer = n>=m? 1:0;
            else answer = n<=m? 1:0;
        }else{
            if(ineq.equals(">")) answer = n>m? 1:0;
            else answer = n<m? 1:0;
        }
        return answer;
    }
}

File: 2st/Riyeon/조건 문자열.py
def solution(ineq, eq, n, m):
    answer = 0
    if ineq == '<':
        if eq == '=':
            answer = 1 if n <= m else 0
        else:
            answer = 1 if n < m else 0
    elif eq == '=':
        answer = 1 if n >= m else 0
    else:
        answer = 1 if n > m else 0
    return answer